README.md

```diff
@@ -69,158 +68,131 @@ B) **Rendering in Twig** is shown below in the [Rendering Pagerfantas](#renderin
 
-$adapter = new DoctrineORMAdapter($queryBuilder);
-$pagerfanta = new Pagerfanta($adapter);
-
-return $this->render('@YourApp/Main/example.html.twig', [
-    'my_pager' => $pagerfanta,
-]);
+final class MyController extends AbstractController
+{
+    public function pagerfanta(): Response
+    {
+        $adapter = new DoctrineORMAdapter($queryBuilder);
+        $pagerfanta = new Pagerfanta($adapter);
+
+        return $this->render(
+            '@YourApp/Main/example.html.twig',
+            [
+                'my_pager' => $pagerfanta,
+            ]
+        );
+    }
+}
 ```
 
-You then call the the Pagerfanta Twig extension, passing in the Pagerfanta instance.
-The routes are generated automatically for the current route using the variable "page" to propagate the page number.
-By default, the bundle uses the *DefaultView* with the *default* name. The default syntax is:
+You then call the the `pagerfanta` function from the Twig extension, passing in the Pagerfanta instance. The routes are generated automatically for the current route using the variable "page" to propagate the page number. By default, the bundle uses the `Pagerfanta\View\DefaultView` class to render the pager.
 
 ```twig
-<div class="pagerfanta">
-    {{ pagerfanta(my_pager) }}
-</div>
+{{ pagerfanta(my_pager) }}
 ```
 
-By default, the "page" variable is also added for the link to the first page. To 
-disable the generation of `?page=1` in the url, simply set the `omitFirstPage` option
-to `true` when calling the `pagerfanta` twig function:
+By default, the "page" variable is also added for the link to the first page. To disable the generation of `?page=1` in the URL, set the `omitFirstPage` option to `true` when calling the `pagerfanta()` Twig function.
 
 ```twig
-<div class="pagerfanta">
-    {{ pagerfanta(my_pager, 'default', { 'omitFirstPage': true}) }}
-</div>
+{{ pagerfanta(my_pager, 'default', {'omitFirstPage': true}) }}
 ```
 
-You can omit the template parameter to make function call shorter, in this case the
-default template will be used:
+You can omit the template parameter to make function call shorter, in this case the default template will be used.
 
 ```twig
-<div class="pagerfanta">
-    {{ pagerfanta(my_pager, { 'omitFirstPage': true }) }}
-</div>
+{{ pagerfanta(my_pager, {'omitFirstPage': true }) }}
 ```
 
-If you have multiple pagers on one page, you'll need to change the name of the `page` parameter.
-Here's an example:
+If you are using a parameter other than `page` for pagination, you can set the parameter name by using the `pageParameter` option when rendering the pager.
 
 ```twig
-<div class="pagerfanta">
-    {{ pagerfanta(my_other_pager, 'default', {'pageParameter': '[page_other]'}) }}
-</div>
+{{ pagerfanta(my_pager, 'default', {'pageParameter': '[other_page]'}) }}
 ```
 
-Note the square brackets around `page_other`; this won't work without them.
+Note that the page parameter *MUST* be wrapped in brackets (i.e. `[other_page]`).
```

Comment:
From what I can tell it traces back to https://github.com/whiteoctober/WhiteOctoberPagerfantaBundle/pull/45 and is essentially a "this is how the PropertyPath class works" thing.

Reply:
The string could just be wrapped in brackets before being used. That would both make PropertyPath happy and leave this bit of config simpler. This could even be optional, to make the migration from white-october bundle easier. Shall I file a new issue?

Reply:
Yeah, that'd be a good thing to look at.  It's been a behavior deeply rooted in the bundle for ages and would help the DX a bit if that hard requirement could go away.

Reply:
That's #5 now.